Sources/Utils/SDKVersion.swift

```diff
@@ -17,4 +17,4 @@
 /// Do not edit this field.
```

Comment:
It says - "Do not edit this field" :) It'll be auto-updated while preparing release.

Reply:
oh thanks for pointing that out 👍